website/src/pages/en/supported-networks.mdx

```diff
@@ -10,7 +10,7 @@ import { useI18n } from '@/i18n'
 
 export const getStaticProps = getSupportedNetworksStaticProps
 
-<Heading.H2 id="supported-networks" className="mb-4 text-24 text-white">
+<Heading.H2 id="supported-networks" className="mb-4">
```

Comment:
@hayderkg – I hadn't noticed this code before. I think we should replace it with `## Supported Networks` which will automatically be translatable and hopefully the right size... if it's too small, then I guess `# Supported Networks` would be the right size, though I think we have an ESLint rule that disallows H1s in Markdown because it should be set in the frontmatter. Which it is, but we're not showing it because of `hideContentHeader: true`... actually, maybe we can just remove that, or does that look wrong?